packages/playwright/src/index.ts

```diff
@@ -10,6 +10,8 @@ function getDeviceConfig() {
   const env: Record<string, any> = {
     ...process.env,
     RECORD_ALL_CONTENT: 1,
+    // it doesn't log anything eagerly but it makes it possible to enable verbose logs with DEBUG=pw:browser
+    RECORD_REPLAY_VERBOSE: 1,
```

Comment:
What's the added overhead from this added logging

Reply:
I doubt it's high because there are really not that many logs produced by this verbose output from what I can tell. Writing to stdout shouldn't incur a significant overhead and the data logged there is probably available anyway. Just to be sure, we can ask @Domiii what he thinks about this.

Reply:
I don't have any numbers on log overhead, but I would intuitively agree that this should not be too bad. Having numbers would be better though 😅